main/src/libraries/wrdac/include/wrdac/clients/icubClient.h

```diff
@@ -292,6 +293,31 @@ namespace wysiwyd{
             */
             bool push(const yarp::sig::Vector &target, const yarp::os::Bottle &options = yarp::os::Bottle(),std::string sName="target");
 
+
+            bool pushKarmaLeft(const std::string &objName, const double &targetPosYLeft,
+                               const std::string &armType = "selectable",
+                               const yarp::os::Bottle &options = yarp::os::Bottle());
+
+            bool pushKarmaRight(const std::string &objName, const double &targetPosYRight,
+                                const std::string &armType = "selectable",
+                                const yarp::os::Bottle &options = yarp::os::Bottle());
+
+            bool pushKarmaFront(const std::string &objName, const double &targetPosXFront,
+                                const std::string &armType = "selectable",
+                                const yarp::os::Bottle &options = yarp::os::Bottle());
+
+            bool pushKarmaLeft(const yarp::sig::Vector &objCenter, const double &targetPosYLeft,
+                               const std::string &armType = "selectable",
+                               const yarp::os::Bottle &options = yarp::os::Bottle(), const std::string &sName="target");
```

Comment:
It is not clear what `sName` is in this context? Can you please add (brief) documentation for those methods?